pkg/controllers/chart/chart.go

```diff
@@ -51,11 +52,26 @@ const (
 )
 
 var (
-	commaRE              = regexp.MustCompile(`\\*,`)
-	deletePolicy         = metav1.DeletePropagationForeground
-	DefaultJobImage      = "rancher/klipper-helm:v0.8.2-build20230815"
-	DefaultFailurePolicy = FailurePolicyReinstall
-	defaultBackOffLimit  = pointer.Int32(1000)
+	commaRE                   = regexp.MustCompile(`\\*,`)
+	deletePolicy              = metav1.DeletePropagationForeground
+	DefaultJobImage           = "rancher/klipper-helm:v0.8.2-build20230815"
+	DefaultFailurePolicy      = FailurePolicyReinstall
+	defaultBackOffLimit       = pointer.Int32(1000)
+	defaultPodSecurityContext = &corev1.PodSecurityContext{
```

Comment:
Same request - put an empty line above these to avoid unnecessary whitespace changes on the previous lines.